resources/views/index.blade.php

```diff
@@ -3,7 +3,32 @@
 ])
 
 @section('content')
-
+    <div class="container-fluid mt-4 p-4" style="background-color: #0049ff; ">
+        <div class="row ps-4">
+            <div class="col m-2 ad-card">
+                <h2 class="text-uppercase mt-5 p-5 ad-text">
+                    De nieuwste producten die onze adverteerders aanbieden
+                </h2>
```

Comment:
Waar is deze tekst? ik zie alleen een groot blauw vak?

Reply:
![afbeelding](https://github.com/Gieljermo/Bazaar/assets/25012362/775ee6ed-fdff-434d-b784-9aabc1541a30)

Reply:
![image](https://github.com/Gieljermo/Bazaar/assets/104079236/3b007f32-3e26-4c06-bfd9-17b691bb99cd)
het is voor mij zo. Wat heb je gedaan dat te krijgen?